### PR TITLE
fix nans in naive kl divergence kernel introduced by div by 0.

### PR DESCRIPTION
**Pull request by mdoijade**
-- This address issue https://github.com/rapidsai/raft/issues/716
-- remove redundant commented code as well.

### CODE REVIEW

File: cpp/test/distance/distance_base.cuh [hunk 1]
```diff
@@ -450,10 +449,6 @@ class DistanceTest : public ::testing::TestWithParam<DistanceInputs<DataType>> {
     }
     naiveDistance(
       dist_ref.data(), x.data(), y.data(), m, n, k, distanceType, isRowMajor, metric_arg, stream);
-    //    size_t worksize = raft::distance::getWorkspaceSize<distanceType, DataType, DataType,
```
Comment (cjnolet) [inline cpp/test/distance/distance_base.cuh:453]: If this is unneeded, can it just be removed?

**Review by cjnolet — CHANGES_REQUESTED** [review]

File: cpp/test/distance/distance_base.cuh (hunk 1 above)
Comment (mdoijade) [reply cpp/test/distance/distance_base.cuh:453]: this is commented code which I've already removed.

Comment (cjnolet) [reply cpp/test/distance/distance_base.cuh:453]: Got it! Wow, I totally misread the diff.

**Review by cjnolet — APPROVED** [review]

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**